prometheus/collectors/collectors.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package collectors provides several collectors that implement
+// prometheus.Collector.
```

Comment:
That's a bit of a stutter and doesn't say so much. How about the following:

Package collectors provides implementations of prometheus.Collector to conveniently collect process and Go-related metrics.